static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -114,12 +112,8 @@ export const init = (): Promise<void> => {
             });
         });
 
-        const gptPath = isInVariantSynchronous(commercialGptPath, 'variant')
-            ? '//securepubads.g.doubleclick.net/tag/js/gpt.js'
-            : config.get('libs.googletag');
-
         // Just load googletag. Prebid will already be loaded, and googletag is already added to the window by Prebid.
-        return loadScript(gptPath, { async: false });
+        return loadScript(config.get('libs.googletag'), { async: false });
```

Comment:
Can we have a fallback here in case this config is missing?

Reply:
I've just restored this to the original implementation, however if the config prop 'libs.googletag' doesn't exist (it's feasible it could accidentally be removed) the consequences would be no ads at all, so yeah I'll add the fallback as suggested.